README.md

```diff
@@ -495,7 +521,15 @@ Save output to file, defaults to STDOUT
 
 #### Android build fails on Windows for no good reason
 
-Both npm and android file hierarchies tend to get very deep and even worse when you combine them. Since Windows file system has a max length, long file name addresses will result in numerous errors including `Execution failed for task ':react-native-vector-icons:processReleaseResources'`. So try to keep the path to your project folder as short as possible. 
+Both npm and android file hierarchies tend to get very deep and even worse when you combine them. Since Windows file system has a max length, long file name addresses will result in numerous errors including `Execution failed for task ':react-native-vector-icons:processReleaseResources'`. So try to keep the path to your project folder as short as possible.
+
+#### Wrong icons are shown after upgrading this package
+
+You probably didn't update the font files linked to your native project after upgrading. You can relink the project or you can manually update them.
```

Comment:
This only applies to android, iOS doesn't copy it. This is easily fixed with the gradle approach, maybe worth mentioning?